Dashboard/extract.py

```diff
@@ -0,0 +1,119 @@
+"Extract Script: Pulls product information from ASOS webspage"
+import json
+import logging
+import requests
+from bs4 import BeautifulSoup
+
+DEFAULT_TIMEOUT = 30
```

Comment:
Good use of global constant, could be better if given with units and with where it is used
```suggestion
DEFAULT_REQUEST_TIMEOUT_SECONDS = 30
```